sql/updates/latest-dev.sql

```diff
@@ -94,3 +94,33 @@ GRANT SELECT ON _timescaledb_catalog.chunk_id_seq TO PUBLIC;
 GRANT SELECT ON _timescaledb_catalog.chunk TO PUBLIC;
 
 -- end recreate _timescaledb_catalog.chunk table --
+
+-- First we update the permissions of the compressed hypertables to
+-- match the associated hypertable.
+WITH
+    hypertables AS (
+	SELECT format('%I.%I', ht.schema_name, ht.table_name)::regclass AS hypertable_oid,
+	       format('%I.%I', ct.schema_name, ct.table_name)::regclass AS compressed_hypertable_oid
+	FROM _timescaledb_catalog.hypertable ht
+	JOIN _timescaledb_catalog.hypertable ct ON ht.compressed_hypertable_id = ct.id
+    )
+UPDATE pg_class
+   SET relacl = (SELECT relacl FROM pg_class WHERE oid = hypertable_oid)
+  FROM hypertables
+ WHERE oid = compressed_hypertable_oid;
+
+-- Now we update the permissions of chunks of both compressed and
+-- uncompressed hypertables to match the permissions of the associated
+-- hypertable.
+WITH
+    chunks AS (
+        SELECT format('%I.%I', ht.schema_name, ht.table_name)::regclass AS hypertable_oid,
+               format('%I.%I', ch.schema_name, ch.table_name)::regclass AS chunk_oid
+          FROM _timescaledb_catalog.hypertable ht
+	  JOIN _timescaledb_catalog.chunk ch ON ht.id = ch.hypertable_id
+	  WHERE NOT ch.dropped
```

Comment:
Do we really want `WHERE NOT ch.dropped` here? Won't that create a problem for `pg_dump` that should also back-up all dropped chunks as well? Could this result in having pointers to chunks that don't exist?

Reply:
No. The dropped chunks do not exist, they are just present in the catalog, so if we include them we get an error about missing chunks when running update.